frontend/pages/configurator.vue

```diff
@@ -638,12 +638,41 @@ const duplicateItem = (index) => {
   testItems.value.push(newItem); // Neues Item zu `testItems` hinzufügen
 };
 
-const deleteItem = (index) => {
-  const itemId = testItems.value[index].id;
+const deleteItem = async (index) => {
+  const item = testItems.value[index];
+  const itemId = item.id;
+
+  // Prüfen, ob die Item-Config eine persistente ID aus der Datenbank hat
+  if (itemId && !item.isUnsaved) {
+    try {
+      const token = localStorage.getItem("token");
+      await $fetch(`http://localhost:8000/api/item_configs/${itemId}`, {
```

Comment:
The URL has to contain the BACKEND_URL variable otherwise it will not work in production (maybe this is also not the case for the other?)

Reply:
Done. Please check line 332. I added "http://localhost:8000"; as an OR Statement. When I dont use that some functions do not work and I dont understand why since I Import the exact same URL out of env. Maybe its the usual problem with my system. Please check if it works for you.